cosmos/launchpad/account.go

```diff
@@ -15,7 +19,23 @@ func (l Launchpad) AccountBalance(ctx context.Context, request *types.AccountBal
 		return nil, ErrNodeConnection
 	}
 
+	height, err := strconv.ParseInt(resp.Height, 10, 64)
```

Comment:
Can you please change the corresponding `swagger.yml` prop to `integer` instead and regenerate the client?

Reply:
done by 96ca7e1